index.bs

```diff
@@ -986,7 +986,8 @@ for="PublicKeyCredential" method>\[[CollectFromCredentialStore]](origin, options
 
 <div link-for-hint="PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)">
 
-This [=internal method=] accepts three arguments:
+{{PublicKeyCredential}}'s [=interface object=]'s implementation of the 
+<dfn for="PublicKeyCredential" method>\[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)</dfn> [=internal method=] accepts three arguments:
```

Comment:
let's please not put this long unnecessary sentence back and find another place for the \<dfn>.  apologies for not cleaning this up myself earlier...  am looking into it....